src/entrypoint/graphql.ts

```diff
@@ -72,14 +72,15 @@ const resolvers = {
   Query: {
     me: async (_, __, { uid, user }) => {
       const { phone, username, contacts, language, level } = user
```

Comment:
we can't do it that way? `const { phone, username, contacts, language, level, twoFactorEnabled } = user`? we should I think

Reply:
I refactored this a bit to change how twoFactorEnabled is being determined. 

But did you mean we should add another field (`twoFactorEnabled`) directly to the schema?